automation/src/test/scala/org/broadinstitute/dsde/test/api/SnapshotAPISpec.scala

```diff
@@ -326,6 +370,30 @@ class SnapshotAPISpec extends AnyFreeSpecLike with Matchers with BeforeAndAfterA
     }
   }
 
+  private def getDataset(datasetName: String, projectId: String, authToken: AuthToken)(implicit executionContext: ExecutionContext) = {
+    import org.typelevel.log4cats.Logger
+    import org.typelevel.log4cats.slf4j.Slf4jLogger
+
+    implicit lazy val contextShift: ContextShift[IO] = cats.effect.IO.contextShift(executionContext)
+    implicit lazy val timer: Timer[IO] = cats.effect.IO.timer(executionContext)
+    implicit val logger: org.typelevel.log4cats.StructuredLogger[IO] = Slf4jLogger.getLogger[IO]
+
+    val bqService = GoogleBigQueryService.resource[IO](
+      GoogleCredentials.create(new AccessToken(authToken.value, null)),
+      Blocker.liftExecutionContext(executionContext),
+      GoogleProject(projectId)
+    )
+
+    bqService.use {
+      bq =>
+        for {
+          dataset <- bq.getDataset(datasetName)
+        } yield {
+          dataset
+        }.toList
+    }.unsafeRunSync()
```

Comment:
up to you: you could simplify this to:
```
bqService.use(_.getDataset(datasetName)).unsafeRunSync()
```
which returns an `Option[Dataset]`; in the test you could check to see if the Option is nonEmpty. Or, add the `.toList` like you have now and check its size